File: lab-1/domain/band.py
from domain.artist import Artist


class Band(Artist):
    def __init__(self, name, debut_date, genre, country):
        super().__init__(name, debut_date)
        self.genre = genre
        self.country = country
        self.albums = []

    def introduce(self):
        print(f"Hello, we are {self.name}, a {self.genre} band from {self.country}.")
        print(f"We have been activating in the music industry since {self.debut_date}.")

File: src/main/kotlin/performer/implementations/artist/implementations/Band.kt
package performer.implementations.artist.implementations
import media_item.implementations.Album
import performer.implementations.artist.Artist

class Band(
        name: String,
        debutDate: String,
        val genre: String,
        val country: String
) : Artist(name, debutDate) {
    val albums = mutableListOf<Album>()

    override fun introduce() {
        println("Hello, we are $name, a $genre band from $country.")
        println("We have been activating in the music industry since $debutDate.")
    }

    override fun toString(): String {
        return "Band(name='$name', debutDate='$debutDate', genre='$genre', country='$country')"
    }

}